modules/serverless-kafka-sasl.adoc

```diff
@@ -0,0 +1,83 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-configuring-sasl-authentication-against-an-apache-kafka_{context}"]
+== Configuring SASL authentication
+
+.Prerequisites
+
+* A username and password for the Kafka cluster.
+* Choose the SASL mechanism to use, for example `PLAIN`, `SCRAM-SHA-256` or `SCRAM-SHA-512`.
+* If TLS is enabled, you also need the `ca.crt` certificate file for the Kafka cluster.
+
+Red Hat recommends to enable TLS in addition to SASL.
```

Comment:
What do you think about changing this to a `NOTE`? Do you think it's important enough to also add to the end of the assembly introduction?